sphinx/dev/best_practice.rst

```diff
@@ -130,7 +138,7 @@ exactly what code they are supposed to be reviewing (e.g. by linking to the
 relevant branches on GitHub).
 
 Do not merge to the ``master`` branch until the reviewer is satisfied with the
-work. It is not required to agree with or implement every suggestion the
+work and the `integration test`_ passes. It is not required to agree with or implement every suggestion the
```

Comment:
This link is broken in the [RtD rendering](http://pfs-2d-pipeline.readthedocs.io/en/tickets-pipe2d-83/dev/best_practice.html#workflow). The syntax you're (attempting to use) is a new one on me: it'd be nice if it worked, but it doesn't seem to.